packages/auth/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "module": "ESNext",
```

Comment:
maybe we should specify a target consistent with Headless & Atomic.

Reply:
I'd prefer to do this in a dedicated PR, because atm, Headless, Bueno, and Atomic are using `ESNext`.